src/components/Payments.js

```diff
@@ -16,33 +17,19 @@ function Payments(props) {
         </tr>
       </thead>
       <tbody>
-        <tr>
-          <td>2018-05-12</td>
-          <td>GBP</td>
-          <td>6.89</td>
-          <td>Dinner with friends at a local restaurant</td>
-          <td>Pending</td>
-          <td>
-            <Button>Cancel</Button>
-          </td>
-        </tr>
-        <tr>
-          <td>2018-02-24</td>
-          <td>USD</td>
-          <td>12.23</td>
-          <td>New headphones purchased from Amazon with free delivery</td>
-          <td>Complete</td>
-          <td />
-        </tr>
-        <tr>
-          <td>2017-12-30</td>
-          <td>AUD</td>
-          <td>28.74</td>
-          <td>Groceries for the week</td>
-          <td>Complete</td>
-          <td />
-        </tr>
-      </tbody>
+        {payments.map((payment)=>(<tr>
```

Comment:
inside this `map` function you need to add index to use it for key attribute. Like so:
`(payment, index) => ...`
and in `<tr>` you need to include a `key` -> `<tr key={index}>`.
That is all.